### ISSUE TITLE
username password does not work in mosquitto 1.4.8

### ISSUE THREAD

**Issue by AndroidDesigner** [eclipse/mosquitto#443]
solved!

**Comment by karlp** [comment]
For future reference, it's normally more useful to write down how you solved it, so other people can benefit down the road, rather than deleting all the content of your ticket.  Now, this ticket is pure noise.  Earlier, it could have been pointed to as an example of a common issue with a solution.

**Comment by AndroidDesigner** [comment]
sorry!
this was only my mistake about ubuntu structure and nothing in mosquitto.
you are right karlp!

**Comment by Tifaifai** [comment]
Backup : 
"""
hi.
I have installed mosquitto broker and mosquitto-clients from mosquitto.org correctly.
I used "netstat -at" and see that tcp and tcp6 on 1883 port are listening.
also I can publish and subscribe by mosquitto_pub and mosquitto_sub simple.
Now :
I want to use username/password authentication.so I added this two lines to the mosquitto.conf file in /etc/mosquitto/mosquitto.conf :
password_file /etc/mosquitto/passfile
allow_anonymous false

so that I created passfile by : mosquitto_passwd -c passfile myName
and enter password and passfile created correctly.

then I set mosquitto.conf to mosquitto by : mosquitto -c /etc/mosquitto/mosquitto.conf
Now I expect that publish and subscribe fail without username and password.
BUT PUBLISH and SUBSCRIBE can do their work without username and password yet!!!
why?

"""
